config/fastly/snippets/bad_bot_detection.vcl

```diff
@@ -0,0 +1,140 @@
+  if (
+    req.http.user-agent ~ "^$"
+    || req.http.user-agent ~ "^Java"
+    || req.http.user-agent ~ "^Jakarta"
+    || req.http.user-agent ~ "IDBot"
+    || req.http.user-agent ~ "id-search"
+    || req.http.user-agent ~ "User-Agent"
+    || req.http.user-agent ~ "compatible ;"
+    || req.http.user-agent ~ "ConveraCrawler"
+    || req.http.user-agent ~ "^Mozilla$"
+    || req.http.user-agent ~ "libwww"
+    || req.http.user-agent ~ "lwp-trivial"
+    || req.http.user-agent ~ "PHP/"
+    || req.http.user-agent ~ "urllib"
+    || req.http.user-agent ~ "GT:WWW"
+    || req.http.user-agent ~ "Snoopy"
+    || req.http.user-agent ~ "MFC_Tear_Sample"
+    || req.http.user-agent ~ "HTTP::Lite"
+    || req.http.user-agent ~ "PHPCrawl"
+    || req.http.user-agent ~ "URI::Fetch"
+    || req.http.user-agent ~ "Zend_Http_Client"
+    || req.http.user-agent ~ "http client"
+    || req.http.user-agent ~ "PECL::HTTP"
+    || req.http.user-agent ~ "panscient.com"
+    || req.http.user-agent ~ "IBM EVV"
+    || req.http.user-agent ~ "Bork-edition"
+    || req.http.user-agent ~ "Fetch API Request"
+    || req.http.user-agent ~ "PleaseCrawl"
+    || req.http.user-agent ~ "[A-Z][a-z]{3,} [a-z]{4,} [a-z]{4,}"
+    || req.http.user-agent ~ "layeredtech.com"
+    || req.http.user-agent ~ "WEP Search"
+    || req.http.user-agent ~ "Wells Search II"
+    || req.http.user-agent ~ "Missigua Locator"
+    || req.http.user-agent ~ "ISC Systems iRc Search 2.1"
+    || req.http.user-agent ~ "Microsoft URL Control"
+    || req.http.user-agent ~ "Indy Library"
+    || req.http.user-agent == "8484 Boston Project v 1.0"
+    || req.http.user-agent == "Atomic_Email_Hunter/4.0"
+    || req.http.user-agent == "atSpider/1.0"
+    || req.http.user-agent == "autoemailspider"
+    || req.http.user-agent == "China Local Browse 2.6"
+    || req.http.user-agent == "ContactBot/0.2"
+    || req.http.user-agent == "ContentSmartz"
+    || req.http.user-agent == "DataCha0s/2.0"
+    || req.http.user-agent == "DataCha0s/2.0"
+    || req.http.user-agent == "DBrowse 1.4b"
+    || req.http.user-agent == "DBrowse 1.4d"
+    || req.http.user-agent == "Demo Bot DOT 16b"
+    || req.http.user-agent == "Demo Bot Z 16b"
+    || req.http.user-agent == "DSurf15a 01"
+    || req.http.user-agent == "DSurf15a 71"
+    || req.http.user-agent == "DSurf15a 81"
+    || req.http.user-agent == "DSurf15a VA"
+    || req.http.user-agent == "EBrowse 1.4b"
+    || req.http.user-agent == "Educate Search VxB"
+    || req.http.user-agent == "EmailSiphon"
+    || req.http.user-agent == "EmailWolf 1.00"
+    || req.http.user-agent == "ESurf15a 15"
+    || req.http.user-agent == "ExtractorPro"
+    || req.http.user-agent == "Franklin Locator 1.8"
+    || req.http.user-agent == "FSurf15a 01"
+    || req.http.user-agent == "Full Web Bot 0416B"
+    || req.http.user-agent == "Full Web Bot 0516B"
+    || req.http.user-agent == "Full Web Bot 2816B"
+    || req.http.user-agent == "Fuzz Faster"
```

Comment:
This is a direct compare, but the actual user agent is `Fuzz Faster U Fool` and then a version, so it should be a `~` for partial match eh?

https://github.com/ffuf/ffuf/blob/08ec6bad2aad1e265b20d9036a16a8f30679ed3e/pkg/runner/simple.go#L102

Reply:
Ah, yep - good call! I didn't realize that wasn't the actual user agent. Updated.